transport/tlscommon/tls.go

```diff
@@ -66,7 +66,7 @@ func LoadCertificate(config *CertificateConfig) (*tls.Certificate, error) {
 		return nil, err
 	}
 
-	log.Debugf("Loading certificate: %v and key %v", certificate, key)
+	log.Debugf("Loading certificate: %v", certificate)
```

Comment:
I could remove the certificate logging too if we think it isn't actually useful. We can usually get this by asking for the configuration used to run Beats/Agent/whatever instead although this is more convenient and won't be wrong.

Reply:
I agree with removing it, either here or in another PR. What I believe would be more helpful is to log (as debug) if we're loading a file (and its path) or reading the cert directly from the config.

Reply:
I have a small preference for continuing to log the cert.  Because it is a public key, there shouldn't be any security concerns in logging it and it can make it easy to see things like mistakes in encoding etc.  That being said I'd be OK if it went away.

I really like the idea of logging the location of the key material.

Reply:
There is a helper for detecting a PEM encoded string, so using that I can keep logging the location of the key when it is a file.

Reply:
Also I agree logging the cert isn't an issue.